client/me/help/help-contact/index.jsx

```diff
@@ -376,6 +377,10 @@ const HelpContact = React.createClass( {
 			return <HelpContactConfirmation { ...confirmation } />;
 		}
 
+		if ( olark.isOlarkClosed ) {
```

Comment:
Should this just be checking for olark closure? Would there be cases where olark is closed but support in general is open?